mlir/include/mlir/Dialect/Rock/utility/loweringUtils.h

```diff
@@ -22,6 +22,76 @@ namespace rock {
 struct ConvolutionDims;
 struct GemmSize;
 
+// This is a helper data structure
```

Comment:
This needs more clarity.

Tile of what? What'll might use these outputs? What are K and D? Are they the same for all of these?